docs/source/tutorial.rst

```diff
@@ -0,0 +1,76 @@
+IDAES-GTEP Tutorial Notebook
```

Comment:
Since the tutorial notebook has been updated, do we still want this page in the online documentation? I could update the text to point to the notebook instead of recreating it here.